arff.py

```diff
@@ -164,7 +164,7 @@
 _RE_RELATION     = re.compile(r'^([^\{\}%,\s]*|\".*\"|\'.*\')$', re.UNICODE)
 _RE_ATTRIBUTE    = re.compile(r'^(\".*\"|\'.*\'|[^\{\}%,\s]*)\s+(.+)$', re.UNICODE)
 _RE_TYPE_NOMINAL = re.compile(r'^\{\s*((\".*\"|\'.*\'|\S*)\s*,\s*)*(\".*\"|\'.*\'|\S*)\s*\}$', re.UNICODE)
-_RE_QUOTE_CHARS = re.compile(r'["\'\\ \t%,]')
+_RE_QUOTE_CHARS = re.compile(r'["\'\s%,]', re.UNICODE)
```

Comment:
Do we have a test to ensure this will not capture newlines inappropriately?

Reply:
First, if I understand this correctly, `\s` should capture newlines, right? Second, if `\n` and `\r`are defined as whitespace it should be fine to match them here as this only encodes single values.

Reply:
I think that this is fine on the encoding side. The decoding is however the place which is going wrong (#89).